.travis.yml

```
@@ -13,18 +13,25 @@ cache:
     - ecommerce/static/bower_components
 
 addons:
-    firefox: latest
-    apt:
-        packages:
-            - lcov
+  apt:
+    packages:
+      - lcov
 
 before_install:
-    - docker-compose -f ./.travis/docker-compose-travis.yml up -d
+  - docker-compose -f ./.travis/docker-compose-travis.yml up -d
+
+install:
+  - docker exec -t ecommerce_testing  bash -c 'source /edx/app/ecommerce/ecommerce_env && cd /edx/app/ecommerce/ecommerce/ && make requirements'
 
 script:
-    - docker exec ecommerce_testing /edx/app/ecommerce/ecommerce/.travis/run_tests.sh
+  - docker exec -t ecommerce_testing  bash -c 'source /edx/app/ecommerce/ecommerce_env && cd /edx/app/ecommerce/ecommerce/docs && make html'
```

That's fair, I didn't realize builds were failing silently.